package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-find-rules",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
Yes. Sadly, it looks like I have no permissions to make any changes directly to master (at least as in pushing from my local repo), as it is a protected branch. That's why I currently do all this stuff via the PRs you approve.  
I then include the separate commit for bumping the version within a PR like this, to keep the noise as low as possible for you.

When we have cut a release fixing this particular bug, I'd like to discuss how we can continue with this tool regarding its repo, its permissions, maintain the changelog, how we can make releasing a bit more straightforward, etc.